pkg/kubelet/dockershim/docker_service.go

```diff
@@ -21,17 +21,17 @@ package dockershim
 import (
 	"context"
 	"fmt"
+	"github.com/blang/semver"
```

Comment:
Can you restore imports to how they were before?

Reply:
Sorry,I noticed it.It's correct now.